tests/integration/api/test_api.py

```diff
@@ -128,12 +127,12 @@ def test_post(
         assert response.json()["message"] == "Successfully Updated Object."
         assert response.json()["data"] == test_metadata.dict()
 
-        test_metadata.name = [""]
+        test_metadata.name = []
         response = app.post("/metadata/", data=test_metadata.json())
         assert response.status_code == 422
         assert (
             response.json()["detail"][0]["msg"]
-            == 'string does not match regex "^(?!\\s*$).+"'
+            == "ensure this value has at least 1 items"
```

Comment:
Do we actually care what this error message is? Or can we assume that a 422 is good enough to test this case?